packages/bitcore-node/src/routes/api/tx.ts

```diff
@@ -58,6 +59,19 @@ router.get('/:txId', async (req, res) => {
         tx.receipt[0].gasLimit = evmdata.fvmGasLimit;
         tx.receipt[0].gasPrice = evmdata.fvmGasPrice;
       }
+      if (tx.receipt.length > 0 && tx.receipt[0].log.length > 0) {
```

Comment:
should we filter out non-Transfer events?

Reply:
ok kub